src/views/CreateRecord/Editor.vue

```diff
@@ -52,18 +52,24 @@
           color="primary"
           dark
         >
-          <v-toolbar-title> Edit Record - {{ currentRecord['fairsharingRecord'].name }} </v-toolbar-title>
+          <v-toolbar-title> Edit Record - {{ sections.generalInformation.initialData.metadata.name }} </v-toolbar-title>
           <v-spacer />
 
           <v-btn
             v-for="(panelData) in confirmPanels"
             :id="panelData.name + '_button'"
             :key="panelData.name"
-            class="default ml-2"
+            class="default"
             @click="panelData.show = true"
           >
             {{ panelData.name }}
           </v-btn>
+          <v-btn
+            class="ml-2 default"
+            :href="'#/' + $route.params.id"
+          >
+            Exist editing
```

Comment:
A typo has crept in here (exit -> exist).

Reply:
It's an issue with the link itself, not the code (try existing clicking on standard for instance, it works flawlessly).